run/GEOS/HEMCO_Config.rc

```diff
@@ -145,22 +144,30 @@ Warnings:                    1
     --> hydrophilic OC    :       0.5
     --> fraction POG1     :       0.49
     --> CO to SOAP        :       0.013
-117     Volcano           : on    SO2/SO4
+    --> GFED_subgrid_coag :       false
```

Comment:
This option was missing from the GEOS HEMCO_Config.rc

Reply:
Ok